src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+
+public class ShoppingCartResponseDto {
+    private Long idAndUserId;
+    private List<Long> ticketsId;
+
+    public List<Long> getTicketsId() {
+        return ticketsId;
+    }
+
+    public void setTicketsId(List<Long> ticketsId) {
+        this.ticketsId = ticketsId;
+    }
+
+    public Long getIdAndUserId() {
+        return idAndUserId;
+    }
+
+    public void setIdAndUserId(Long idAndUserId) {
+        this.idAndUserId = idAndUserId;
+    }

```

Comment:
I'm not really like this solution
But it's your decision

Reply:
I try to reduce size of object 
in readmy file I found this: "keep in mind that cartId and userId hold the same value."